module/gurps.js

```diff
@@ -1516,6 +1518,7 @@ Hooks.once('init', async function () {
   GURPS.initiative = new Initiative()
   GURPS.hitpoints = new HitFatPoints()
   GURPS.ConditionalInjury = new GURPSConditionalInjury()
+  GURPS.Quintessence = new GURPSQuintessence()
```

Comment:
Does this need to go in the global namespace? I plan to reduce this as much as possible and rely on imports, if possible.

Reply:
Don't think so, I was just working as I saw CI implemented.